grpc-sys/build.rs

```diff
@@ -23,14 +23,13 @@ mod imp {
 
     pub fn build_or_link_grpc(cc: &mut ::gcc::Config) {
         if let Ok(lib) = ::pkg_config::Config::new()
-            .atleast_version(GRPC_VERSION)
-            .statik(true)
-            .probe("grpc_unsecure") {
+               .atleast_version(GRPC_VERSION)
+               .probe("grpc_unsecure") {
```

Comment:
If static link forces using SSL, we must force use here for dynamic. 

So if users want to link gRPC, they must install all themselves.